.gitlab-ci.yml

```diff
@@ -58,14 +68,6 @@ sniffers:front:
   script:
     - make front-install # Dependencies are required for js imports to pass linters
     - make lintval
-  cache:
-    key:
-      files:
-      # Until https://gitlab.com/gitlab-org/gitlab/issues/118466 lands ...
-        - web/themes/custom/XXX/package.json # ... this path has to be hardcoded
-        - web/themes/custom/XXX/yarn.lock # ... this path has to be hardcoded
-    paths:
-      - ${THEME_PATH}/node_modules/ # Populated during yarn install
```

Comment:
I don't see reason to move this to "only_var_theme" anchor
Can you please explain ?